actors/miner/src/lib.rs

```diff
@@ -1650,17 +1646,21 @@ impl Actor {
                 .map_err(|e| e.wrap("failed to load deadlines"))?;
 
             // Group declarations by deadline, and remember iteration order.
-            let mut decls_by_deadline = HashMap::<u64, Vec<ExpirationExtension>>::new();
+            //
+            // Vec::new() is a const expr, so this operation here is free (well, it'll add 576 bytes
+            // to the compiled binary and that'll get copied to the stack, but that should be pretty
+            // cheap.
+            const EMPTY_VEC: Vec<ExpirationExtension> = Vec::new();
+            let mut decls_by_deadline = [EMPTY_VEC; WPOST_PERIOD_DEADLINES as usize];
```

Comment:
why this way?

Reply:
Well... I wanted to avoid the overhead of using a btreemap when we had a small number of elements anyways.